### PR TITLE
HomePage Updates

### PR DESCRIPTION
**Pull request by BurhanRaja**

### CODE REVIEW

**Comment by BurhanRaja** [comment]
Done I forked it again and updated all. Now, you can do merge easily.

**Comment by Mayank0255** [comment]
@BurhanRaja 
Sorry but found a bug,

In 0 answers also its showing the green badge
image

File: src/services/handleSorting.js [hunk 1]
```diff
@@ -29,12 +40,46 @@ const handleSorting = (sortType, page = '') => {
     case 'Name':
       return (a, b) => a.tagname.localeCompare(b.tagname);
     case 'Username':
-      return (a, b) => a.username.localeCompare(b.username);
+      return (a, b) => a.localeCompare(b.username);
```
Comment (Mayank0255) [inline src/services/handleSorting.js:43]: Please fix this, this is breaking, just revert and it should work

**Comment by BurhanRaja** [comment]
Ok I'll do it. I did the whole thing again maybe that's why it has not changed.

**Comment by BurhanRaja** [comment]
> @BurhanRaja Sorry but found a bug,
> 
> In 0 answers also its showing the green badge image

No worries, will solve the bug

File: src/services/handleSorting.js (hunk 1 above)
Comment (BurhanRaja) [reply src/services/handleSorting.js:43]: It is breaking when Name button is clicked but the other buttons are working.

**Review by Mayank0255 — APPROVED** [review]

**Comment by Mayank0255** [comment]
Merged 🎉🎉

**Comment by BurhanRaja** [comment]
> Merged 🎉🎉

Thank you for the support you have given and also Thank you for not getting angry at me as I made a lot of mistakes. It was my first open source merge and it was a fun experience. Thank you

**Comment by Mayank0255** [comment]
@BurhanRaja 
No issues, keep contributing and growing

**Pull request merged**